### ISSUE TITLE
Environment dependencies' versions

### ISSUE THREAD

**Issue by illorens** [pangeo-data/WeatherBench#8]
Hi, I was trying to get your CNN baseline by running :
> python -m src.train_nn -c src/nn_configs/fccnn_3d.yml

as your README instructs. However, I get the error:
> xarray.core.merge.MergeError: conflicting values for variable 'level' on objects to be combined. You can skip this check by specifying compat='override'.

I think it is an error related to the xarray version. Could you please specify the versions on your environment.yml file?

**Comment by raspstephan** [comment]
I am using xarray=0.13.0. But ideally this shouldn't matter. Could you let me know which xarray you are using and maybe copy the full trace-back of the error. Thanks so much!

**Comment by illorens** [comment]
I'm using xarray 0.15.0. Here's the traceback:

> Traceback (most recent call last):
>   File "/home/illorens/miniconda3/envs/weatherbench/lib/python3.7/runpy.py", line 193, in _run_module_as_main
>     "__main__", mod_spec)
>   File "/home/illorens/miniconda3/envs/weatherbench/lib/python3.7/runpy.py", line 85, in _run_code
>     exec(code, run_globals)
>   File "/mnt/scratch/students/illorens/external_libraries/WeatherBench/src/train_nn.py", line 291, in <module>
>     iterative=args.iterative
>   File "/mnt/scratch/students/illorens/external_libraries/WeatherBench/src/train_nn.py", line 204, in main
>     ds = xr.merge([z, t])
>   File "/home/illorens/miniconda3/envs/weatherbench/lib/python3.7/site-packages/xarray/core/merge.py", line 793, in merge
>     merge_result = merge_core(dict_like_objects, compat, join, fill_value=fill_value)
>   File "/home/illorens/miniconda3/envs/weatherbench/lib/python3.7/site-packages/xarray/core/merge.py", line 555, in merge_core
>     variables, out_indexes = merge_collected(collected, prioritized, compat=compat)
>   File "/home/illorens/miniconda3/envs/weatherbench/lib/python3.7/site-packages/xarray/core/merge.py", line 228, in merge_collected
>     merged_vars[name] = unique_variable(name, variables, compat)
>   File "/home/illorens/miniconda3/envs/weatherbench/lib/python3.7/site-packages/xarray/core/merge.py", line 143, in unique_variable
>     f"conflicting values for variable {name!r} on objects to be combined. "
> xarray.core.merge.MergeError: conflicting values for variable 'level' on objects to be combined. You can skip this check by specifying compat='override'.

I thought it looked like a version issue. It would be great if you included the versions you used in your environment.yml file, please! Thank you so much!

**Comment by raspstephan** [comment]
I changed the merge statement to include `compat='override'`. The error appears because both `geopotential_500` and `temperature_850` contain a level variable but with different values (500 vs 850). This variable will be discarded later anyway, so it save to override it.

This error is independent from the xarray version. I used a slightly different dataset which is why this error did not appear on my side. Thanks a lot for pointing this out!
